content/en/docs/contribute/style/style-guide.md

```diff
@@ -44,22 +44,24 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 ### Use upper camel case for API objects
 
-When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal Case. When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
+When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal case. You may see different capitalization, such as "configMap", in the [API Reference](/docs/reference/kubernetes-api/). When writing general documentation, it's better to use upper camel case, calling it "ConfigMap" instead.
+
+When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
+
+You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence. 
 
 Don't split the API object name into separate words. For example, use
 PodTemplateList, not Pod Template List.
 
-Refer to API objects without saying "object," unless omitting "object"
-leads to an awkward construction.
+The following examples focus on capitalization. Review the related guidance on [Code Style](#code-style-inline-code) for more information on formatting API objects. 
 
-{{< table caption = "Do and Don't - API objects" >}}
+{{< table caption = "Do and Don't - Use Pascal case for API objects" >}}
 Do | Don't
 :--| :-----
-The pod has two containers. | The Pod has two containers.
-The HorizontalPodAutoscaler is responsible for ... | The HorizontalPodAutoscaler object is responsible for ...
```

Comment:
One thing to watch out for: you can find a volume mentioned in Kubernetes in several places, but AIUI _not_ as a resource in the HTTP API. I'm wary in case someone reads the style guide and ends up confused about how storage works.

Reply:
OK @sftim. Can you provide a concrete example?

Reply:
Could remove the "Volume" example. The examples I provided are "tests" of the recommendations.

Reply:
storage volume, volume plugin, a volume or logical drive, volume driver

Reply:
I agree on moving this forward or closing it. 

I'm thinking I'll bring this up again at the next sig docs meeting to ask for synchronous feedback. I'll make a powerpoint slide with the new examples. 

Regarding the volume example, I attempted to literally copy it from the API reference. 

See, for example, 
- https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#volume-v1-core
- this page defines a volume API resource that includes a `hostpath` field.

- https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#podspec-v1-core
- podspec includes a field that is an array of volumes